docs/support_playbook.md

```diff
@@ -63,6 +63,20 @@ To change the accrediting body of a course, you can do the following:
 p = RecruitmentCycle.current.providers.find_by(provider_code: "1YP")
 p.courses.update(accredited_body_code: "1YK")
 ```
+## Transfer courses to another provider
+
+```ruby
+# This can be reviewed in QA before actioning in production
```

Comment:
Do you mean tested/tried in QA before trying in prod? I find the wording a bit confusing.

Reply:
I have updated the wording, thanks